tests/features/field.multilingual.feature

```diff
@@ -6,92 +6,78 @@ Feature: Field Multilingual features
-	When I go to "admin/structure/types/manage/page/fields/field_selectcolor/delete"
-	And I press the "Delete" button
-	Then I should see the success message "The field Select a Color has been deleted from the Basic page content type."
-	When I go to "admin/structure/types/manage/page/groups/group_groupcolor/delete/default"
-	And I press the "Delete" button
-	Then I should see the success message "The group My Group Color has been deleted from the Basic page content type."
+  Scenario: Administrator can translate field and field group labels. Once it is done,
+  users see field and field group labels translated in interface language or in the source language
+  if translations are not done
+    Given I am logged in as a user with the 'administrator' role
+    # Add and translate field and field Group
+    And a field with the following settings is added to "page" type:
+      | Label           | Select a Color  |
+      | Name            | selectcolor     |
+      | Type            | list_text       |
+      | Cardinality     | 1               |
+      | Widget          | options_buttons |
+      | Allowed values  | Red::Blue       |
+      | Translatable    | TRUE            |
+    And I am viewing a multilingual "page" content:
+      | language | title             | field_selectcolor |
+      | en       | Title in English  | Red               |
+      | fr       | Titre en Français | Red               |
+    # Field translation: selectcolor
+    When I go to "admin/structure/types/manage/page/fields/field_selectcolor/translate/fr"
+    And I fill in "edit-strings-fieldfield-selectcolorpagelabel" with "Selectionner une Couleur"
+    And I fill in "edit-strings-fieldfield-selectcolorallowed-valuesred" with "Rouge"
+    And I fill in "edit-strings-fieldfield-selectcolorallowed-valuesblue" with "Bleu"
+    And I press the "Save translation" button
+    Then I should see the success message "3 translations were saved successfully."
+    # Field group creation: groupcolor.
+    When I go to "admin/structure/types/manage/page/display"
+    And I fill in "edit-fields-add-new-group-label" with "My Group Color"
+    And I fill in "edit-fields-add-new-group-group-name" with "groupcolor"
+    And I select "_add_new_group" from "edit-fields-field-selectcolor-parent"
+    And I press the "Save" button
+    Then I should see the success message "Your settings have been saved."
+    Then I should see the success message "New group My Group Color successfully created."
+    # Field group translation: groupcolor.
+    When I go to "content/title-english_en"
+    And I click "New draft"
+    And I select the radio button "Red" with the id "edit-field-selectcolor-en-red"
+    And I select "published" from "edit-workbench-moderation-state-new"
+    And I press the "Save" button
+    Then I should see the success message "Basic page Title in English has been updated."
+    When I go to "content/title-english_fr"
```

Comment:
In fact, it is mandatory in order to allow the translation system registering "My group color" in strings to translate. 
Without this step, the rest of the scenario fails.